Makefile

```diff
@@ -23,3 +23,7 @@ go-mod:
 	export GO111MODULE=on \
 		$(GO) mod tidy && \
 		$(GO) mod verify
+
+.PHONY: verify-go-mod
+verify-go-mod: go-mod
+	hack/tree-status
```

Comment:
This looks good for now. In the future we likely want a separate target that will do all steps that produce any generated code, then run `hack/tree-status`, but I believe `verify-go-mod` is the only place we are doing this for now.

Reply:
Yeah I thought that we may have multiple verification scripts that rely on `hack/tree-status`. For example if we shell format scripts via shfmt.